agent/handlers/task_server_setup_test.go

```diff
@@ -3585,24 +3603,25 @@ func getNetworkBlackHolePortHandlerTestCases(name, fault string, expectedHappyRe
 		"Protocol":    protocol,
 		"TrafficType": trafficType,
 	}
-	happyStateExpectations := func(state *mock_dockerstate.MockTaskEngineState) {
-		task := standardTask()
-		gomock.InOrder(
-			state.EXPECT().TaskARNByV3EndpointID(endpointId).Return(taskARN, true),
-			state.EXPECT().TaskByArn(taskARN).Return(task, true).Times(2),
-			state.EXPECT().ContainerMapByArn(taskARN).Return(containerNameToDockerContainer, true),
-			state.EXPECT().TaskByArn(taskARN).Return(task, true),
-			state.EXPECT().ContainerByID(containerID).Return(dockerContainer, true).AnyTimes(),
-			state.EXPECT().PulledContainerMapByArn(taskARN).Return(nil, true),
-		)
-	}
+
 	tcs := []blackholePortFaultTestCase{
```

Comment:
looks ecs-agent has similar test cases as well?

Reply:
Yeah, the rationale also testing everything here is because the handlers will eventually be hooked up into agent via the `registerFaultHandlers` (i.e. they will be registered once TMDS is initialized)